cloud/README.md

```diff
@@ -0,0 +1,6 @@
+# Cloud xOpera examples
+This folder contains xOpera examples that can be used to deploy services on different popular cloud providers. We try to
+provide simple blueprints that show the creation and usage of different cloud resources with xOpera and Ansible. Since
+these examples are in general more complex, they cannot be deployed directly since they require some prerequisites to be
```

Comment:
_... complex, they require more attention to satisfy the prerequisites._

Reply:
Refactored.